### PR TITLE
🤖 Add .meta/config.json file for concepts

### PR DESCRIPTION
**Pull request by ErikSchierboom**
_TL;DR; Add authors/contributors to a new `concepts/<slug>/.meta/config.json` to Concepts in the same way you have for Concept Exercises._

---

To properly attribute authorship/contributorship for an individual concept, each concept will have its own `.meta/config.json` file with `authors` and `contributors` keys like those for concept and practice exercises. We're also moving the concept's blurb from the track's `config.json` to the `.meta/config.json` file, also similar to concept and practice exercises.

See https://github.com/exercism/docs/pull/96/files for the update to the specification.

This PR adds a `.meta/config.json` file for all concepts found in the track's `config.json` file containing:

- A `blurb` key, which value is copied from the `config.json` file and subsequently removed from the `config.json` file
- An `authors` key, the value of which is copied from the exercise teaching the concept (if any)
- A `contributors` key, the value of which is copied from the exercise teaching the concept (if any)

**For ease, the concept's authors/contributors are pre-populated with the authors/contributors of the exercise that teaches the concept (if any).**

## Maintainers TODO list

These are the things you, the maintainers, should do:

- [ ] Update the authors (if needed) 
- [ ] Update the contributors (if needed)

**For clarity, authors should be the people who originally created the concept documents on this track. Contributors should be people who have substantially contributed to the concept documents. PRs for typos or multiple-concept PRs do not automatically mean someone should be considered a contributor to that concept. Those non-concept-specific contributions will get recognition through Exercism's reputation system in v3.**

You can either update this PR by pushing a new commit to this PR, or merge this and do a follow-up PR later. The former has the disadvantage that once we've added linting rules to `configlet` for the concept `.meta/config.json` file, the configlet CI check will start failing. 

## Tracking

https://github.com/exercism/v3-launch/issues/27

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
Following a suggestion that was made, we've updated the PR to pre-populate the authors/contributors with the authors/contributors of the exercise that teaches the concept (if any). This gives us a nice starting point, as the concept documents used to be exercise documents. You, the maintainers, should check to see if the authors/contributors are correct and add missing authors/contributors.

File: concepts/anonymous-functions/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "blurb": "In addition to named functions organized in modules, Elixir has anonymous functions that can be defined dynamically as needed.",
+  "authors": [
+    "neenjaw"
+  ],
+  "contributors": []
```
Comment (neenjaw) [inline concepts/anonymous-functions/.meta/config.json:6]:
```suggestion
  "contributors": [
    "angelikatyborska"
  ]
```

File: concepts/atoms/.meta/config.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "blurb": "You can use atoms whenever you have a set of constants to express. An atom's value is its own name.",
+  "authors": [
+    "neenjaw"
+  ],
+  "contributors": []
```
Comment (neenjaw) [inline concepts/atoms/.meta/config.json:6]:
```suggestion
  "contributors": [
    "angelikatyborska"
  ]
```

File: concepts/bit-manipulation/.meta/config.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "blurb": "Bitwise operations, such as AND, OR, and XOR, can be performed on integers using functions from the Bitwise module.",
+  "authors": [
+    "neenjaw"
+  ],
+  "contributors": []
```
Comment (neenjaw) [inline concepts/bit-manipulation/.meta/config.json:6]:
```suggestion
  "contributors": [
    "angelikatyborska"
  ]
```

File: concepts/cond/.meta/config.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "blurb": "There are three conditional expressions in Elixir: if, case, and cond. Cond is usually used when there are more than two logical branches and each branch has a condition based on different variables.",
+  "authors": [
+    "neenjaw"
+  ],
+  "contributors": []
```
Comment (neenjaw) [inline concepts/cond/.meta/config.json:6]:
```suggestion
  "contributors": [
    "angelikatyborska"
  ]
```

File: concepts/file/.meta/config.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "blurb": "Functions for working with files are provided by the File module.",
+  "authors": [
+    "angelikatyborska"
+  ],
+  "contributors": []
```
Comment (neenjaw) [inline concepts/file/.meta/config.json:6]:
```suggestion
  "contributors": [
    "neenjaw"
  ]
```

File: concepts/maps/.meta/config.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
+  "blurb": "Maps are a data structure that holds key-value pairs. Keys and values can be of any data type, and the keys must be unique.",
+  "authors": [
+    "neenjaw"
+  ],
+  "contributors": []
```
Comment (neenjaw) [inline concepts/maps/.meta/config.json:6]:
```suggestion
  "contributors": [
    "angelikatyborska"
  ]
```

File: concepts/module-attributes-as-constants/.meta/config.json [hunk 7]
```diff
@@ -0,0 +1,7 @@
+{
+  "blurb": "There are no constants in Elixir. Module attributes may be used to play a similar role.",
+  "authors": [
+    "neenjaw"
+  ],
+  "contributors": []
```
Comment (neenjaw) [inline concepts/module-attributes-as-constants/.meta/config.json:6]:
```suggestion
  "contributors": [
    "angelikatyborska"
  ]
```

**Review by neenjaw — CHANGES_REQUESTED** [review]
@angelikatyborska I think you were missed on the concepts that I authored that I'm pretty sure you had a hand in reviewing.  Any other changes needed?

**Comment by angelikatyborska** [comment]
@neenjaw I have to reject all of your requests about adding me to those concepts 😅 

According to https://github.com/exercism/v3/pull/1335, I was *NOT* involved in the concepts: `anonymous-functions` and `bit-manipulation`

According to https://github.com/exercism/v3/pull/1129, I was *NOT* involved in the concepts: `cond` and `atoms`

According to https://github.com/exercism/v3/pull/1336, I was *NOT* involved in the concepts `maps` and `module-attributes-as-constants`

However, you were involved in `file` (https://github.com/exercism/v3/pull/2958) and I pushed a commit to mark that in both the concept's and the exercise's config.

**Review by angelikatyborska — APPROVED** [review]

**Comment by neenjaw** [comment]
Weird. I don't remember who did have a hand in those PRs if not you...

**Review by neenjaw — APPROVED** [review]
